auth/server/src/main/java/com/seagates3/controller/UserPolicyController.java

```diff
@@ -111,6 +113,38 @@ ServerResponse detach() throws DataAccessException {
     return serverResponse;
   }
 
+ public
+  ServerResponse list() throws DataAccessException {
+    String pathPrefix = requestBody.get("PathPrefix");
+    String userName = requestBody.get("UserName");
+
+    LOGGER.info("List policies for user: " + userName);
+    ServerResponse serverResponse = null;
+
+    try {
+      User user = userDAO.find(requestor.getAccount().getName(), userName);
```

Comment:
This is not required. As we will be having entire user object inside requestor.- requestor.getUser()
After authorization code merge, this need to be changed

Reply:
Okay. This will be taken care after Authorization code merge.

Reply:
we can retain this method as it is and add one more wrapper method
void  find( User user, String name)  {
      find(user.getName(), name);       // ->  find(String, String)
}

Reply:
Let Authorization code call wrapper method find(User, String)

Reply:
@bkirunge7 is it okay if we do this change once the authorization code is merged? because this method is written to get the user and user is not added to the Requestor yet. So this will be done only after authorization code is merged.